Documentation/guides/k8s-aws-ansible/install.rst

```diff
@@ -0,0 +1,1430 @@
+    192.168.96.0/19
+
+SERVICE\_IP\_RANGE 
+
+::
+
+    192.168.192.0/19 
+
+K8S\_SERVICE\_IP 
+
+::
+
+    192.168.192.1/19
+DNS\_SERVICE\_IP 
+
+::
+
+    192.168.192.10/19
+
+CILIUM 
+
+::
+
+    10.X.X.Y/8
+
+1. Getting started with AWS
+----------------------------
+
+1.1 Create new vpc and network
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium$ mkdir installation;cd installation/
+    core@host:~/cilium$ aws ec2 create-vpc --cidr-block 192.168.32.0/19
+    core@host:~/cilium/installation$ export VPC_ID=vpc-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 create-subnet --vpc-id $VPC_ID --cidr-block 192.168.32.0/19
+    core@host:~/cilium/installation$ export SUBNET_ID=subnet-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 modify-subnet-attribute --subnet-id $SUBNET_ID --map-public-ip-on-launch
+    core@host:~/cilium/installation$ aws ec2 create-internet-gateway
+    core@host:~/cilium/installation$ export GATEWAY_ID=igw-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 attach-internet-gateway --vpc-id $VPC_ID --internet-gateway $GATEWAY_ID
+    core@host:~/cilium/installation$ aws ec2 create-route-table --vpc-id $VPC_ID
+    core@host:~/cilium/installation$ export ROUTE_TABLE_ID=rtb-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 associate-route-table --subnet-id $SUBNET_ID --route-table-id $ROUTE_TABLE_ID
+    core@host:~/cilium/installation$ aws ec2 create-route --route-table-id $ROUTE_TABLE_ID --destination-cidr-block 0.0.0.0/0     --gateway-id $GATEWAY_ID
+    core@host:~/cilium/installation$ aws ec2 modify-vpc-attribute --vpc-id=$VPC_ID --enable-dns-support
+    core@host:~/cilium/installation$ aws ec2 modify-vpc-attribute --vpc-id=$VPC_ID --enable-dns-hostnames
+
+1.2 Configuring Key Pair and Security Group 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium/installation$ aws ec2 create-key-pair --key-name key --query 'KeyMaterial' --output text > key.pem
+    core@host:~/cilium/installation$ chmod 400 key.pem
+    core@host:~/cilium/installation$ aws ec2 create-security-group --group-name SG --description SG --vpc-id $VPC_ID
+    core@host:~/cilium/installation$ export SECURITY_GROUP_ID=sg-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 authorize-security-group-ingress --group-id $SECURITY_GROUP_ID --protocol tcp --port 22     --cidr x.x.x.x/8
+    core@host:~/cilium/installation$ aws ec2 authorize-security-group-ingress --group-id $SECURITY_GROUP_ID --protocol tcp --port 22     --cidr x.x.x.x/32
+    core@host:~/cilium/installation$ aws ec2 authorize-security-group-ingress --group-id $SECURITY_GROUP_ID --source-group     $SECURITY_GROUP_ID  --protocol all --port all
+
+1.3 Spinning up the VMs  
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium/installation$ export IMAGE_ID=ami-xxxxxxxx
+    core@host:~/cilium/installation$ aws ec2 describe-images --image-ids ami-xxxxxxxx
+
+
+1.3.1 CREATE MASTERS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
I assume that the `XXXXXXXXXXXXXXXXXXXXXXXXXXXX` needs to be substituted? Must be mentioned otherwise people will just copy & paste.

What is `${PRIVATE_IPV4}` set to?